src/qt/overviewpage.cpp

```diff
@@ -302,7 +302,8 @@ void OverviewPage::setClientModel(ClientModel *model)
         // Show warning, for example if this is a prerelease version
         connect(model, &ClientModel::alertsChanged, this, &OverviewPage::updateAlerts);
         updateAlerts(model->getStatusBarWarnings());
-        setMonospacedFont(false);
+        connect(model->getOptionsModel(), &OptionsModel::useEmbeddedMonospacedFontChanged, this, &OverviewPage::setMonospacedFont);
+        setMonospacedFont(model->getOptionsModel()->getUseEmbeddedMonospacedFont());
```

Comment:
<s>does it really need to be configured via settings?

Too many settings for users is not always better :)
I'd suggest to just enable it for everyone</s>

Is it backported code, right?